cl_tf2/src/cl-tf2/stamped-data.lisp

```diff
@@ -71,6 +71,7 @@
    - define a class 'number-stamped' with slots 'num-value' and 'header'
    - define a constructor-function 'make-number-stamped'
    - define a function 'copy-number-stamped'
+   - define a function 'copy-ext-number-stamped'
```

Comment:
This was the best name I could think of. Maybe we can change `copy-...` to be a method?
Then, we can have multiple `copy-...` functions with different signatures. Of course, then the syntax is not being displayed in the log window anymore.. or is it?

Anyway, having it as an overloaded method somehow feels better than this `-ext-`-stuff. I'll try that out when I have time.